tests/sentry/sentry_apps/test_sentry_app_updater.py

```diff
@@ -253,3 +258,109 @@ def test_delete_service_hook_on_update(self) -> None:
         updater.run(self.user)
         with assume_test_silo_mode(SiloMode.REGION):
             assert len(ServiceHook.objects.filter(application_id=internal_app.application_id)) == 0
+
+    @with_feature("organizations:service-hooks-outbox")
+    def test_update_service_hooks_with_outbox_feature_enabled(self) -> None:
+        """Test _update_service_hooks when organizations:service-hooks-outbox feature is enabled."""
+        # Create installation
+        installation = self.create_sentry_app_installation(
+            slug=self.sentry_app.slug, organization=self.org, user=self.user
+        )
+
+        with outbox_runner():
+            # Clear any existing outbox entries
+            ControlOutbox.objects.all().delete()
```

Comment:
Why are we deleting all control outboxes here? Could we alternatively flush them all first, before running our checks after?